packages/react/src/disclosure/disclosure.tsx

```diff
@@ -0,0 +1,223 @@
+import { ChevronDown } from '@obosbbl/grunnmuren-icons-react';
+import { filterDOMProps } from '@react-aria/utils';
+import { type VariantProps, cva, cx } from 'cva';
+import {
+  type ForwardedRef,
+  type HTMLAttributes,
+  type HTMLProps,
+  type RefAttributes,
+  createContext,
+  useContext,
+  useId,
+  useRef,
+} from 'react';
+import { useDisclosure } from 'react-aria';
+import {
+  Button,
+  ButtonContext,
+  type ButtonProps,
+  type ContextValue,
+  DisclosureContext,
+  DisclosureGroupStateContext,
+  Provider,
+  type DisclosureProps as RACDisclosureProps,
+  useContextProps,
+} from 'react-aria-components';
+import { useDisclosureState } from 'react-stately';
+
+const disclosureButtonVariants = cva({
+  base: 'inline-flex cursor-pointer items-center rounded-lg outline-none data-[focus-visible]:outline-focus',
+  variants: {
+    size: {
+      // Ensure a minimum click area of 44x44px, while making it look like it only has the size of the content
+      dense: '-m-2.5 p-2.5 data-[focus-visible]:outline-offset-[-0.625rem]',
+      // Ensure a minimum click area of 44x44px, and a minimum size of 44x44px
+      regular: 'min-h-11·min-w-11·data-[focus-visible]:outline-offset',
+    },
+    withChevron: {
+      true: '[&[aria-expanded="true"]_svg]:rotate-180',
+      false: null,
+    },
+    /**
+     * When the button is without text, but with a single icon.
+     * @default false
+     */
+    isIconOnly: {
+      true: '[&>svg]:h-7 [&>svg]:w-7',
+      false: 'gap-2.5',
+    },
+  },
+  defaultVariants: {
+    size: 'regular',
```

Comment:
Kom på at `size` er `dense` i alle eksemplene, kanskje det skulle vært default i stedenfor? @alexanbj 
```suggestion
    size: 'dense',
```